src/condor_starter.V6.1/jic_local.cpp

```diff
@@ -326,58 +325,13 @@ JICLocal::initUserPriv( void )
 		return true;
 	}
 
-	char* owner = NULL;
-	if( job_ad->LookupString( ATTR_OWNER, &owner ) != 1 ) {
-		dprintf( D_ALWAYS, "ERROR: %s not found in JobAd.  Aborting.\n", 
-				 ATTR_OWNER );
-		return false;
-	}
-
-#ifdef WIN32
-		// we only care about or expect to find this attribute if
-		// we're on windoze...
-	char* domain = NULL;
-	if( job_ad->LookupString( ATTR_NT_DOMAIN, &domain ) != 1 ) {
-		dprintf( D_ALWAYS, "ERROR: %s not found in JobAd.  Aborting.\n", 
-				 ATTR_NT_DOMAIN );
+	if (!init_user_ids_from_ad(*job_ad)) {
+		dprintf(D_ERROR, "Failed to intialize user_priv, aborting.\n");
 		return false;
 	}
 
-	if( ! init_user_ids(owner,domain) ) { 
-		dprintf( D_ALWAYS,
-				 "ERROR: Bad or missing credential for user \"%s@%s\"\n",
-				 owner, domain );
-	} else {  
-		rval = true;
-		dprintf( D_FULLDEBUG, "Initialized user_priv as \"%s@%s\"\n", 
-				 owner, domain );
-	}
-	if( domain ) {
-		free( domain );
-		domain = NULL;
-	}
-
-#else /* UNIX */
-
-	if( ! init_user_ids_quiet(owner) ) { 
```

Comment:
no quiet option (linux code path)

Reply:
I don't think we need the quiet option here. The user account not existing would be very unusual for local universe, so some extra errors logged by init_user_ids_from_ad() will not excessively pollute the log.